test/integration/post.api.tests.js

```diff
@@ -94,16 +94,63 @@ describe(`post api`, function () {
 
   // TODO: "should add a track from hot tracks"
 
-  // TODO: "should add a track to an existing playlist"
+  it('should add a track to an existing playlist', async function () {
+    const postWithPlaylist = post;
+    postWithPlaylist.pl = { id: 'create', name: randomString() };
 
-  // TODO: should add a track to a new playlist (where ? if this isnot the bookmarklet ?)
+    const { body } = await util.promisify(api.addPost)(jar, postWithPlaylist);
+    const pId = body._id;
+    const name = body.name;
+    const ctx = 'bk';
+    const playlist = body.pl;
+    const description = 'this is a description';
+
+    await new Promise((resolve, reject) =>
```

Comment:
suggestion non bloquante pour réduire un peu la verbosité de cette opération: utiliser `appPost()` de `api-client.js` (cf https://github.com/openwhyd/openwhyd-solo/blob/73734c0ab665f6701af7aa8b5b9ce635ad8a2b2f/test/api-client.js#L133)

(à moins que l'usage de cette requête soit volontaire)

Reply:
En fait je comprends pas pourquoi tu fais cette requête en plus de l'appel à `api.addPost`, plus haut.
Est-ce que l'intention de ce test est d'éditer une track déjà postée, ou reposter cette track ?

Reply:
C'est le seul moyen que j'ai trouvé pour créer une playlist préexistante. Est-ce qu'il y a un moyen de créer une playlist sans passer par un `addPost` ?

Après je réplique 'should edit a track's name' que vous aviez écrit https://github.com/openwhyd/openwhyd-solo/commit/08bcc1ac83e37cefc54efe1227e4f2d6e60e5689#diff-7a5275a7d5f1fe903c14384760be14979f2064dee2ba244c96a7986fc43fa614R19.